tests/users/test_database_model.py

```diff
@@ -45,7 +44,7 @@ def test_second_user_cannot_be_admin(self):
         user = UserModel.query.filter_by(email="user1@email.com").first()
         self.assertTrue(user is not None)
         self.assertTrue(user.id is not None)
-        self.assertTrue(user.name == "User1")
+        self.assertTrue(user.name == "UserA")
```

Comment:
What's the reason for changing name from `User1` to `UserA`?

Reply:
User1 is not a valid name. refer https://github.com/anitab-org/mentorship-backend/blob/ee009cfe6c45a1c88375b3c30787c030692ca417/app/utils/validation_utils.py#L4